src/main/java/me/moodcat/eemcsdata/JsonData.java

```diff
@@ -0,0 +1,68 @@
+package me.moodcat.eemcsdata;
+
+import lombok.Data;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * A data class to store the JSON elements in.
+ */
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class JsonData {
```

Comment:
This is not just `JsonData`, it's the json format of the AcousticBrains result. Change the classname.